index.html

```diff
@@ -359,11 +358,13 @@ <h3>
                     relevant settings object</a>, or <code>null</code> if
                     unavailable.
                   </li>
-                  <li>Invoke <a>Start Register</a> with <var>scope</var> and
-                  <var>src</var> members of the <var>registration</var>, a new
-                  <var>promise</var>, <var>client</var>, <a>manifest URL</a>,
-                  plus the <var>type</var> and <var>use_cache</var> members of
-                  the <var>registration</var>,
+                  <li>Invoke <a href=
```

Comment:
I already have a PR out for this: #792. It's a lot more extensive than what's here (you can't just invoke Register with those arguments).

Can you just remove this change and I'll land that PR later?

Reply:
I’ll just kill this PR. It was against `master` anyway and I think it should have been against `gh-pages`.